### PR TITLE
Make tests more flexible, accommodate changed CrossEntropyLoss signature

### PR DESCRIPTION
**Pull request by bhancock8**

### CODE REVIEW

File: metal/end_model/em_defaults.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
     'show_plots': True,
 
     # Network
-    'batchnorm': True,
+    'batchnorm': False,
```
Comment (ajratner) [inline metal/end_model/em_defaults.py]: Anything weird going on here :)?

File: metal/end_model/loss.py [hunk 2]
```diff
@@ -7,33 +7,33 @@ class SoftCrossEntropyLoss(nn.Module):
 
     Args:
         weight: a tensor of relative weights to assign to each class.
-        size_average:
-        reduce:
+        reduction:
```
Comment (jdunnmon) [inline metal/end_model/loss.py]: Do we want a descriptor here?

File: metal/end_model/em_defaults.py (hunk 1 above)
Comment (bhancock8) [reply metal/end_model/em_defaults.py]: Haha, good check. This gets at a deeper question we should discuss sometime. For most applications (including our lstm_tests), with batchnorm on you do better; we want to make it "easy to do the right," which may mean leaving it on by default. But there's also something to be said for not adding in widgets that people don't know about, so default to no regularization, no dropout, no batchnorm, etc. until they explicitly add it, so that we know they're aware of it.

File: metal/end_model/loss.py (hunk 2 above)
Comment (bhancock8) [reply metal/end_model/loss.py]: Good point. Added documentation.

File: metal/end_model/em_defaults.py (hunk 1 above)
Comment (jdunnmon) [reply metal/end_model/em_defaults.py]: I think as long as we document well what the defaults are, we're fine.  I'd argue for having it on but having a set of docs that explicate this very clearly.  We could even have a "raw" config and a "default" config, where the "raw" version has no bells/whistles and "default" has our opinion of best default practices.
